Formula/kim-api-v2.rb

```diff
@@ -0,0 +1,34 @@
+class KimApiV2 < Formula
+  desc "The Knowledgebase of Interatomic Models (KIM) API"
+  homepage "https://openkim.org"
+  url "https://s3.openkim.org/kim-api/kim-api-v2-2.0.0.txz"
+  sha256 "beb9eb47cfe9ac9a4ad22528da89a67415a5b033075a9318a5d4a800dd4a3f3d"
+
+  depends_on "doxygen" => :build
+  depends_on "cmake" => :build
+  depends_on "gcc" => :build
```

Comment:
Why is GCC needed? And why build-only? It seems weird.

Reply:
The package uses c, c++, and fortran compilers to build.  I wasn't sure if gcc was needed as a runtime dependency.  From your question, I would now guess that it is...

Also, some runtime uses of the tools provided by this package call cmake and the compilers to work correctly.  However, not all users would need to use these tools.  (In other systems I would split these tools and the header files, into a separate 'dev' package.)  So, I'm not sure if cmake should also be listed as a runtime dependency.  What is the homebrew philosophy on this?

Reply:
Yes, if Fortran code is included, GCC will be needed at runtime.
```suggestion
  depends_on "gcc" # for gfortran
```